Prox/Prox/PlaceDetails/PlaceDetailsCardView.swift

```diff
@@ -186,19 +188,57 @@ class PlaceDetailsCardView: UIScrollView {
         if frame != newFrame { frame = newFrame }
     }
 
+    // TODO: don't duplicate PlaceCarousel code
+    func updateUI(forPlace place: Place) {
```

Comment:
Oops! I'll keep that in mind. I've been thinking in a react-like style (UI state is rendered directly from program state) but perhaps that loses its value when you're mutating the UI, rather than declaring it as a function.

fwiw, I haven't done any MVC beyond the theory so I'm likely to be messy.